### PR TITLE
New service of FakeExchange adapter

### PR DESCRIPTION
**Pull request by AndreyAndreev**
FakeExchange implements an API of ExchangeAdapter (SpotController), while just keeping all order books in memory and assumes that every user have enough balance for its orders.

### CODE REVIEW

File: client/Lykke.Service.FakeExchange.Client/FakeExchangeClient.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using Lykke.HttpClientGenerator;
+
+namespace Lykke.Service.FakeExchange.Client
+{
+    public class FakeExchangeClient : IFakeExchangeClient
+    {
+        //public IControllerApi Controller { get; }
```
Comment (PeGilo) [inline client/Lykke.Service.FakeExchange.Client/FakeExchangeClient.cs:7]: Commented lines

File: src/Lykke.Service.FakeExchange.Services/Lykke.Service.FakeExchange.Services.csproj [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <AssemblyName>Lykke.Service.FakeExchange.Services</AssemblyName>
+    <RootNamespace>Lykke.Service.FakeExchange.Services</RootNamespace>
+    <Version>1.0.0</Version>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Lykke.Common" Version="7.0.1" />
```
Comment (PeGilo) [inline src/Lykke.Service.FakeExchange.Services/Lykke.Service.FakeExchange.Services.csproj]: TC requires 7.0.3 now

File: client/Lykke.Service.FakeExchange.Client/FakeExchangeClient.cs (hunk 1 above)
Comment (AndreyAndreev) [reply client/Lykke.Service.FakeExchange.Client/FakeExchangeClient.cs:7]: It's from the template, `Lykke.Service.FakeExchange.Client` and `Lykke.Service.FakeExchange.AzureRepositories` projects are staying in the initial state for now

File: src/Lykke.Service.FakeExchange.Core/Domain/OrderBook.cs [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+        public OrderBook(
+            string pair,
+            IBalancesService balancesService)
+        {
+            Pair = pair;
+            _balancesService = balancesService;
+        }
+        
+        public void Add(Order order)
```
Comment (PeGilo) [inline src/Lykke.Service.FakeExchange.Core/Domain/OrderBook.cs:64]: Nice method )

**Review by PeGilo — DISMISSED** [review]

File: src/Lykke.Service.FakeExchange.Services/Lykke.Service.FakeExchange.Services.csproj (hunk 2 above)
Comment (AndreyAndreev) [reply src/Lykke.Service.FakeExchange.Services/Lykke.Service.FakeExchange.Services.csproj]: Updated

**Review by PeGilo — APPROVED** [review]

**Review by aushakov — COMMENTED** [review]
Delete settings.json
Change setting format to yaml
Update solution according latest template

**Pull request merged**